backend/006-meal-plan.sql

```diff
@@ -36,6 +36,12 @@ create or replace function app.set_meal_plan_new_person_id() returns trigger as
   end;
   $$ language plpgsql;
 
+-- returns set of meal plan tags
```

Comment:
I noticed that you have overwritten an existing sql. You should create a seperate migration under migrations folder as a seperate sql file. Only then will the migrations run. You need to `docker-compose down` Set the env variable COMPOSE_FILE and then build docker again.

Reply:
Done. Thanks for your help! :)